### DISCUSSION TITLE
Receiving error message when tries to connect to MS SQL db via java - using Mirth 4.4.2

### DISCUSSION THREAD

**Discussion by lilianamSF** [nextgenhealthcare/connect#6364] [Q&A]
Hello,

We need assistance on troubleshooting the error message below. Has anyone seen this error?
Thank you.

ERROR  (com.mirth.connect.connectors.jdbc.DatabaseReceiverScript:192): An error occurred while polling for messages, retrying after 10000 ms... java.lang.NullPointerException at com.mirth.connect.connectors.jdbc.DatabaseReceiverScript.poll(DatabaseReceiverScript.java:188) at com.mirth.connect.connectors.jdbc.DatabaseReceiver.poll(DatabaseReceiver.java:140) at com.mirth.connect.donkey.server.channel.PollConnectorJob.execute(PollConnectorJob.java:49) at org.quartz.core.JobRunShell.run(JobRunShell.java:213) at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:557)

**Comment by ab-mg-23** [comment]
The NPE is likely because your script is returning a null value. The call to `result.toString()` at line 188 of the below linked code should pop a NPE if that is the case.

https://github.com/nextgenhealthcare/connect/blob/e0b3a89a30b576a5ef091ec7023df80e6da068ee/server/src/com/mirth/connect/connectors/jdbc/DatabaseReceiverScript.java#L175-L189
